pkg/tfvars/libvirt/cache.go

```diff
@@ -102,6 +102,26 @@ func cacheKey(etag string) (key string, err error) {
 	return hex.EncodeToString(hashed[:]), nil
 }
 
+type progressWatcher struct {
+	total    uint64
+	oldTotal uint64
+}
+
+func (w *progressWatcher) Write(p []byte) (int, error) {
+	n := len(p)
+	w.total += uint64(n)
+	// Log progress every 3 MiB
```

Comment:
the image size as of today is around 700 MB, for each 3MB there's going to be 233 debug lines.... that's not acceptable.

Reply:
@abhinavdahiya How about a time-based approach, where log messages are printed every 5 seconds IFF > 3MiB has been read?

Reply:
Anything more than an couple of lines for download progress is _imo_ more than what's acceptable.